packages/doenetml-worker-rust/lib-doenetml-core/src/core/props/prop_view.rs

```diff
@@ -112,6 +112,7 @@ mod into_prop_with_meta {
                 value: prop.value.into(),
                 came_from_default: prop.came_from_default,
                 changed: prop.changed,
+                node: None,
```

Comment:
I originally had it as `origin`, but then it was set in the same place where `origin` referred to the data query it came from so I was worried about confusion. Since I couldn't come up with another name, I just made it `node`. I'd be fine changing it back to `origin` if you think there would be no confusion.

Reply:
Yeah, I think it won't be confusing. They type will indicate the difference :-)

Reply:
OK, changing it back to `origin`